dockerfiles/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM eclipse-temurin:17-jre-jammy as flyway
+##################################################
+# Step 1 - unpack Flyway
+##################################################
+FROM eclipse-temurin:17-jre-jammy 
```

Comment:
> [!Note]
> I originally picked this image to ensure that users were exactly the same between the build image and the production image.
>
> Now that I look at this a second time, I see that we're not changing ownership of any files, so that shouldn't actually matter.  We could use a much smaller image like `alpine` here, but since the actual `flyway` image also uses this as its base, this is more efficient from a layers-downloaded perspective because this doesn't add any additional layers that we weren't already downloading for the `flyway` stage.